stream.go

```diff
@@ -414,8 +420,10 @@ func (cs *clientStream) newAttemptLocked(isTransparent bool) (*csAttempt, error)
 	method := cs.callHdr.Method
 	var beginTime time.Time
 	shs := cs.cc.dopts.copts.StatsHandlers
+	nameResolutionDelayed := false
+	nameResolutionDelayed = cs.nameResolutionDelayed
 	for _, sh := range shs {
-		ctx = sh.TagRPC(ctx, &stats.RPCTagInfo{FullMethodName: method, FailFast: cs.callInfo.failFast})
+		ctx = sh.TagRPC(ctx, &stats.RPCTagInfo{FullMethodName: method, FailFast: cs.callInfo.failFast, NameResolutionDelay: nameResolutionDelayed, IsTransparentRetry: isTransparent})
```

Comment:
how is isTransparent getting calculated?

Reply:
This is being passed as an argument of newAttempt method everytime there was a transparent retry.
https://github.com/grpc/grpc-go/blob/master/stream.go#L405